packages/venia-concept/package.json

```diff
@@ -24,14 +24,16 @@
     "start:debug": "node --inspect-brk ./node_modules/.bin/webpack-dev-server --progress --color --env.mode development",
     "test": "jest src",
     "validate-queries": "node ./validate-queries.js",
-    "watch": "webpack-dev-server --progress --color --env.mode development"
+    "watch": "webpack-dev-server --progress --color --env.mode development",
+    "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook:build": "build-storybook -c .storybook -o storybook-dist"
```

Comment:
Please keep the alphabetical order of this list 😉

Reply:
https://github.com/zetlen/eslint-plugin-package-json 👀